README.md

```diff
@@ -6,5 +6,10 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+
+Rails exists to help us build web applications. As a back-end app, it allows the ability to do things such as authentication (register/login) and managing data (storing/saving, sending, etc)
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
```

Comment:
The server, controller and model parts of your diagram are correct. The role played by the view is a bit fuzzy in your diagram; insofar as we've discussed it, the `view` is not an agent in and of itself, but rather a passive template that gets used by the controller; the controller populates that template with data and sends the finished result back to the Server.